runtime/compiler/il/J9SymbolReference.cpp

```diff
@@ -277,7 +277,7 @@ SymbolReference::getTypeSignature(int32_t & len, TR_AllocationKind allocKind, bo
                        else
                           {
                           int32_t numParens = 0;
-                          while (s && (s[0] == '[') && (s[1] == 'L'))
+                          while (s && (s[0] == '[') && (s[1] == 'L'))//should we add "|| s[1] == 'Q'"?
```

Comment:
A few days ago, I would have said yes - but now from Tobi's recent explanations, I understand array elements will be references instead, so you shouldn't add a condition for `Q` here.